### DISCUSSION TITLE
Current Projects in the Profile

### DISCUSSION THREAD

**Discussion by SporyTike** [RetroAchievements/RAWeb#795] [Ideas]
At the moment I use my user motto to let people know, what my current developer projects are. But this way is a bit annoying because you need to copy the game ID and insert it into the URL to find out which game is behind the ID. That's why I would like to have a way to show the games directly in my profile with a link to the game page (like in the example). 

My idea would be to add an extra section in the settings for user with developer rank or higher. The section would be similar to the "Similar Games" section on a game page with a line to add a game ID and a dropdown with up to eight saves and a remove button to delete a game ID.

![image](https://user-images.githubusercontent.com/45672834/80162558-f7310500-85d3-11ea-8512-53fc650adabf.png)

**Comment by televandalist** [comment]
Would be nice to tie into handling reservations on-site.

**Comment by jeffjenx** [comment]
Would you be satisfied with a DYI implementation, like a WYSIWYG editor for a profile description?

I saw another issue talking about forum signatures and thought it might be better to allow users to create their own custom description with code snippets such as links and text formatting. Similar to the motto, but more functional.

This would benefit all users who wish to expand about themselves on their own profile and avoid creating features that are specific to individual user groups.

**Comment by Jamiras** [comment]
Implemented in #1026 

![image](https://user-images.githubusercontent.com/16140035/172312000-f0ce5fd3-c1c2-45db-82be-ed4a02795465.png)
